src/components/organisms/SubscribeSection/Form/index.tsx

```diff
@@ -22,7 +22,7 @@ const Form: React.FC = () => {
   return (
     <form
       className={styles.form}
-      action="https://sweedom.us10.list-manage.com/subscribe/post?u=a08bf93caae4063c4e6a351f6&amp;id=24c0ecc49a"
+      action="https://dvc.us10.list-manage.com/subscribe/post?u=a08bf93caae4063c4e6a351f6&amp;id=24c0ecc49a"
```

Comment:
let's double check that it still works this way and people are being subsribed to the right list

Reply:
ping @jendefig :)

Reply:
I just tested with my hotmail from cml.dev. It did get added to the DVC Newsletter list.  There isn't a CML list.  So do we need one or the DVC Newsletter is what's expected?

Reply:
good point, idk if we want to remove this completely since we don't have a CML mailing list.

Reply:
we decided for now to keep it the same list for simplicity, I think it's not part of this PR to change this :)